index.html

```diff
@@ -1,70 +1,443 @@
+          </header>
+          <div class="item-parrent">
+            <div class="item1">
+              <p class="conopy">CONOPY</p>
+            </div>
+            <div class="item2"></div>
+            <div class="item3">
+              <p class="back">Back End Dev</p>
+            </div>
+            <div class="item4"></div>
+            <div class="item5">
+              <p class="year">2015</p>
+            </div>
+          </div>
+          <div class="text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="tags">
+            <ul class="ul">
+              <li class="html">
+                <p class="h">HTML</p>
+              </li>
+              <li class="css">
+                <p class="c">CSS</p>
+              </li>
+              <li class="js">
+                <p class="j">JavaScript</p>
+              </li>
+            </ul>
+          </div>
+
+          <div class="action">
+            <div class="boxx">
+              <button class="see-btn">
+                <div class="see">
+                  <p class="se">See Project</p>
+                </div>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="deatails2">
+        <div class="shot4"></div>
+        <div class="bo">
+          <header class="pro-title2">
+            <h2>Multi-Post Stories</h2>
+          </header>
+          <div class="item-parrent">
+            <div class="item1">
+              <p class="conopy">CONOPY</p>
+            </div>
+            <div class="item2"></div>
+            <div class="item3">
+              <p class="back">Back End Dev</p>
+            </div>
+            <div class="item4"></div>
+            <div class="item5">
+              <p class="year">2015</p>
+            </div>
+          </div>
+          <div class="text">
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+          </div>
+          <div class="tags">
+            <ul class="ul">
+              <li class="html">
+                <p class="h">HTML</p>
+              </li>
+              <li class="css">
+                <p class="c">CSS</p>
+              </li>
+              <li class="js">
+                <p class="j">JavaScript</p>
+              </li>
+            </ul>
+          </div>
+
+          <div class="action">
+            <div class="boxx">
+              <button class="see-btn">
+                <div class="see">
+                  <p class="se">See Project</p>
+                </div>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="footer">
+      <div class="top">
+        <div class="about">
+          <h1>About Myself</h1>
+        </div>
```

Comment:
- [ ] The 'About Myself' title is styled correctly, but it should use a different `h` tag than the `h1` tag. The `h1` tag is used for the main title of the page and should be one but no more than one `h1` tag per page for better SEO purposes. Kindly use an `h2` or `h3` tag in this section title. Please [read this article](https://www.reliablesoft.net/h1-tag/) for more information.